src/lib/api.js

```diff
@@ -32,6 +32,15 @@ app.use((req, res, next) => {
     next();
 });
 
+/**
+ * Middleware that converts uncaught Errors to failureJson
+ */
+app.use((err, req, res, next) => {
```

Comment:
Does express check the parameter count of the middleware to determine whether to pass the `err`? If so, that's neat.